src/testdom/nodes.ts

```diff
@@ -482,6 +482,9 @@ export class ServiceCommandTest extends ServiceMemberTestNode {
     override updateState() {
         if (this.options?.hasChildren) super.updateState()
     }
+    public setOutput(s: string) {
```

Comment:
why do we need this?

Reply:
i made the output setter public, not needed.

Reply:
it was protected.

Reply:
![image](https://user-images.githubusercontent.com/10767146/164336250-b284abf7-a496-41bf-a35c-685c88bde9dd.png)

Reply:
you can set test.status, but not test.output

Reply:
You need to merge main, you will have merge issues. output is public in main.